spec/lib/myfinance/resources/receivable_account_spec.rb

```diff
@@ -0,0 +1,85 @@
+require "spec_helper"
+
+describe Myfinance::Resources::ReceivableAccount do
+  describe "#create", vcr: true do
+    let(:params) { { due_date: '2015-08-15', amount: 150.99 } }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.